### PR TITLE
Integrate WTF into zone.js

### PR DESCRIPTION
**Pull request by mhevery**
/cc: @IgorMinar @btford @vicb

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,4 @@ blink-core/
 blink-idl/
 parsed-idl/
 /node_modules
+.idea
```
Comment (vicb) [inline .gitignore]: -1 you should add it to your own global gitignore

Comment (IgorMinar) [inline .gitignore]: nah.. this line is to protect us from Misko not to protect Misko from our commits.

I'm fine with keeping it in.

**Comment by mhevery** [comment]
I have updated all with your comments. It fails on IE9, but I don't know how to get it reproduced locally. Could anyone help.

**Comment by mhevery** [comment]
I found the reason why IE9 was failing.

File: lib/patch/functions.js [hunk 2]
```diff
@@ -1,96 +1,100 @@
...
-        var args = bindArgs(arguments);
-        id = delegate.apply(obj, args);
-        ids[id] = true;
-        return id;
+var wtf = require('../wtf');
+var Zone = require('../core').Zone;
+
+function patchSetClearFunction(window, fnNames) {
+  fnNames.forEach(function (setName) {
+    var repeating = setName == 'setInterval';
```
Comment (vicb) [inline lib/patch/functions.js]: the caller should pass the info

Comment (mhevery) [inline lib/patch/functions.js]: fixed

**Pull request closed without merging**